pkg/server/commands/listusers/list_users_rpc_test.go

```diff
@@ -691,19 +693,43 @@ func TestListUsersCycles(t *testing.T) {
 			model: `model
 			schema 1.1
 			type user
-
-			type folder
-			relations
-				define viewer: [user, document#viewer]
-
 			type document
```

Comment:
How does this test pass? The model is invalid in the first place.

Reply:
Oh I see, we added a warn and still run it.

See https://github.com/openfga/openfga/pull/1451#discussion_r1532383675